Refresher/Info.plist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleIdentifier</key>
+	<string>org.littlebig.refresh</string>
```

Comment:
shouldnt this be `org.littlebigrefresh.refresher`? or do these ids not work like that

Reply:
I think it would be `com.littlebigrefresh.refresher` actually; I've always assumed they're reverse FQDN.

Reply:
Yeah - I set it to a dummy value to get these comments and be sure.